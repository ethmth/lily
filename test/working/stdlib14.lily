
let a::int = [6,75,8,73,1,42,1]

a.mergesort()

print(a)
let b::int = [2 ,3, 4]

let c:: int = [1, 5, 7]
#print(merge(b,c))