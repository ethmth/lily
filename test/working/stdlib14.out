[1, 1, 6, 8, 42, 73, 75]
